### PR TITLE
Integrate new logo

### PR DESCRIPTION
**Pull request by elnappo**
image
image

### CODE REVIEW

File: src/nsupdate/templates/base.html [hunk 1]
```diff
@@ -94,7 +95,7 @@
                             </ul>
                         </li>
                     </ul>
-                </div>_blac
```
Comment (ThomasWaldmann) [inline src/nsupdate/templates/base.html]: oops?

**Comment by elnappo** [comment]
Also added the logo to `README.rst`.

File: README.rst [hunk 2]
```diff
@@ -1,5 +1,4 @@
-About nsupdate.info
-===================
+.. image:: logo/logotype_blue.png
```
Comment (ThomasWaldmann) [inline README.rst]: please no README.rst changes - we use this from setup.py and i suspect this won't work for pypi.org.

**Comment by elnappo** [comment]
removed logo from readme

**Comment by ThomasWaldmann** [comment]
thanks!

**Pull request merged**